.github/workflows/coverage.yaml

```diff
@@ -5,7 +5,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [8, 10, 12]
```

Comment:
maybe add `v14` ?

is the `v8` incompatibility due to the testing tooling or?

Reply:
Added Node 14. Yeah removed 8 because it was not supported by `yargs`